accord-core/src/main/java/accord/local/Commands.java

```diff
@@ -91,7 +102,7 @@ private Commands()
     {
     }
 
-    public enum AcceptOutcome { Success, Redundant, RejectedBallot, Truncated }
+    public enum AcceptOutcome { Success, Redundant, RejectedBallot, Retired, Truncated }
```

Comment:
It's used to refine the handling of the state machine for command stores that no longer own a range. This helps more clearly support a change that ensures prompt forward progress (it would happen eventually, but might be delayed without this change). I will add a comment explaining the rationale. This is primarily used in PreAccept to skip certain PreAccept steps that might not work if the command has been erased due to the range being retired.